<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="error_label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите корректные данные</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="input_data">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame, QLabel, QToolTip {
    border: 2px solid #191970;
    border-radius: 4px;
    padding: 2px;
    background-color: #B0C4DE;
	color: #000000;
}</string>
          </property>
          <property name="text">
           <string>Входные данные:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="output_data">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame, QLabel, QToolTip {
    border: 2px solid #191970;
    border-radius: 4px;
    padding: 2px;
    background-color: #B0C4DE;
	color: #000000;
}</string>
          </property>
          <property name="text">
           <string>Выходные данные:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="sex_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame, QLabel, QToolTip {
    border: 2px solid #191970;
    border-radius: 4px;
    padding: 2px;
    background-color: #B0C4DE;
	color: #000000;
}</string>
              </property>
              <property name="text">
               <string>Пол:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sex_input">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #191970;
    border-radius: 10px;
    padding: 0 8px;
    background: #B0C4DE;
    selection-background-color: darkgray;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="height_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame, QLabel, QToolTip {
    border: 2px solid #191970;
    border-radius: 4px;
    padding: 2px;
    background-color: #B0C4DE;
	color: #000000;
}</string>
              </property>
              <property name="text">
               <string>Рост:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="height_input">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #191970;
    border-radius: 10px;
    padding: 0 8px;
    background: #B0C4DE;
    selection-background-color: darkgray;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="weight_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame, QLabel, QToolTip {
    border: 2px solid #191970;
    border-radius: 4px;
    padding: 2px;
    background: #B0C4DE;
}</string>
              </property>
              <property name="text">
               <string>Вес:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="weigth_input">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #191970;
    border-radius: 10px;
    padding: 0 8px;
    background: #B0C4DE;
    selection-background-color: darkgray;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="thin_coeff">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame, QLabel, QToolTip {
    border: 2px solid #191970;
    border-radius: 4px;
    padding: 2px;
    background: #B0C4DE;
}</string>
          </property>
          <property name="text">
           <string>Коэффициент похудения</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="calculate_btn">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

    background-color: #B0C4DE;
    border: 3px solid #191970;
    border-radius: 10px;
}
QPushButton:hower {
    color: 7B00F0;
}
QMainWindow {
    background-color: #D195FF;
    color: #D195FF;
    background-image: url(&quot;C:\Users\Oleg\PycharmProjects\ProjectPyQtкартинки&quot;);
    background-repeat: no-repeat;
    background-position: 0, 0, 100, 200 px;
}s</string>
        </property>
        <property name="text">
         <string>Посчитать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="quit_btn">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #B0C4DE;
    border: 3px solid #191970;
    border-radius: 10px;
}
QPushButton:hower {
    color: 778899;
}
QMainWindow {
    background-color: #778899;
    color: #778899;
    background-repeat: no-repeat;
    background-position: 0, 0, 100, 200 px;
}s</string>
      </property>
      <property name="text">
       <string>QUIT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
